.github/workflows/run-tests.yml

```
@@ -0,0 +1,92 @@
+
+jobs:
+  flake8:
+    name: Lint nav
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Set up Python ${{ matrix.python-version }}
```

actionlint (1.7.7)

property "python-version" is not defined in object type {}

(expression)